rfd/0006-reverse-tunneling.md

```diff
@@ -0,0 +1,39 @@
+---
+authors: Adam Enger (adamenger@gmail.com)
+state: discussion
+---
+
+# RFD 6 - Reverse Tunneling
```

Comment:
Adam, thanks for the PR and writing the doc

Reply:
💯 👍